locales/fix-message-catalogs.js

```diff
@@ -0,0 +1,46 @@
+//@ts-check
+
+/**
+ * There doesn't seem to be any easy way to have TypeScript
+ * ignore first-party code that is explicitly imported by
+ * code that is type-checked, so this script goes through
+ * all lingui's messages catalog JS files and explicitly
+ * tells TypeScript to ignore them.
+ */
```

Comment:
Blerg, I really wish I didn't have to lift this script from #497 but I can't seem to find any other way of telling TypeScript to ignore Lingui's compiled message catalogs (because they're imported by our TS code, TypeScript insists on type-checking them even if we tell it to exclude the directory they're in).